src/main/java/org/folio/rest/impl/FiscalYearsApi.java

```diff
@@ -22,13 +22,20 @@
 public class FiscalYearsApi implements FinanceFiscalYears {
 
   private static final String FISCAL_YEARS_LOCATION_PREFIX = getEndpoint(FinanceFiscalYears.class) + "/%s";
+  private static final int FISCAL_YEAR_LENGTH = 4;
 
   @Validate
   @Override
-  public void postFinanceFiscalYears(String lang, FiscalYear entity, Map<String, String> headers,
+  public void postFinanceFiscalYears(String lang, FiscalYear fiscalYear, Map<String, String> headers,
       Handler<AsyncResult<Response>> handler, Context ctx) {
     FiscalYearsHelper helper = new FiscalYearsHelper(headers, ctx, lang);
-    helper.createFiscalYear(entity)
+
+    // series should always be present
+    if (isEmpty(fiscalYear.getSeries())) {
```

Comment:
It is always a system calculated field, even if it was specified in the request, it's better to calculate than to use it

Reply:
I'm calculating and setting series on line 36, is there a better way to calculate?

Reply:
is Empty check is unnecessary , calculate the series always

Reply:
Got it. changed.